test/tests-distributed.cpp

```diff
@@ -223,6 +223,61 @@ TEST(distributed, cuda_pummaMM) {
   f.close();
 }
 
+TEST(distributed, tblis_pummaMM) {
```

Comment:
As I mentioned on slack, lets add a test using TBLIS for a higher order tensor contraction instead of mat-mul. I can help you set it up on Wednesday. A neat thing to do would be to generate code for a completely un-scheduled version, vs the TBLIS scheduled version, and then in the driver application we can assert that the two give identical results (on some sort of random data).

Reply:
Added a test with `C["abij"] = A["acik"] * B["cbkj"]` instead, as discussed.